spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jms/activemq/ActiveMQConnectionFactoryFactory.java

```diff
@@ -79,7 +81,7 @@ private <T extends ActiveMQConnectionFactory> T doCreateConnectionFactory(Class<
 
 	private <T extends ActiveMQConnectionFactory> T createConnectionFactoryInstance(Class<T> factoryClass)
 			throws InstantiationException, IllegalAccessException, InvocationTargetException, NoSuchMethodException {
-		String brokerUrl = determineBrokerUrl();
+		String brokerUrl = this.connectionDetails.getBrokerUrl();
```

Comment:
In addition to the broker URL, I think that `user` and `password` should come from the connection details as well, otherwise it won't be possible to use connected to an ActiveMQ service that requires authentication.